### DISCUSSION TITLE
16

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#930] [Comments]
# 16

구글에서 Git에 관해 검색하면 대부분이 Git 명령어 혹은 EGIT (이클립스의 Git 플러그인) 결과가 나온다.GUI 환경에 이미 익숙한 윈도우 유저들에게는 사실 명령어로 툴을 다루는것이 어색한데,이런 사용자들을 위해 Atlassian (JIRA를 만든 그곳) 에서 GIt GUI 툴인 Sourcetree를 만들었다.(Max os와 window만 지원한다. 리눅스는ㅠ) 굳이 Git을 사용하는데 Sourcetree가 필요한가? IDE (이클립스, 인텔리J 등)에서 제공하는 플러그인으로 사용하면 되지 않나?(하물며 Sublime Text에서도 Git 플러그인을 제공한다)라는 반응을 가질수도 있는데 내가 생각하기에는 2가지 이유로 플러그인 대신에 Source tree를 사용해야 한다. 1. 플러그인에 비해 ..

https://jojoldu.tistory.com/16

**Comment by dkGithup2022** [comment]
왕감사합니다 : )
